### PR TITLE
downgrade logging upgrade cluster

### PR DESCRIPTION
**Pull request by jcantrill**
use 4.15 cluster for logging upgrade job

cc @cahartma @Clee2691

### CODE REVIEW

**Comment by jcantrill** [comment]
/pj-rehearse pull-ci-openshift-cluster-logging-operator-master-upgrade

File: ci-operator/config/openshift/cluster-logging-operator/openshift-cluster-logging-operator-master.yaml [hunk 1]
```diff
@@ -170,11 +170,11 @@ tests:
     architecture: amd64
     cloud: aws
     labels:
-      region: us-east-1
+      region: us-east-2
```
Comment (cahartma) [inline ci-operator/config/openshift/cluster-logging-operator/openshift-cluster-logging-operator-master.yaml:173]: We don't have any pools in us-east-2 btw.   Not sure the impact of changing this label

Comment (cahartma) [reply ci-operator/config/openshift/cluster-logging-operator/openshift-cluster-logging-operator-master.yaml:173]: I'm wrong.  nvm, 4.16 is in us-east-1.   disregard

**Pull request merged**